rust/platform/BUILD

```diff
@@ -0,0 +1,7 @@
+package(default_visibility = ["//visibility:public"])
+
+exports_files(["triple_mappings.bzl"])
```

Comment:
fyi these are for skydoc which is currently broken

Reply:
I think this decl is also necessary in order for the bzl file to have public visibility. Something about how "not mentioning" a bzl file makes it not exist to the rest of the file tree. I can't recall.

Reply:
iirc it suffices to have an empty BUILD file

Reply:
Ack: Removing